pkg/kubelego/watch.go

```diff
@@ -61,7 +107,7 @@ func (kl *KubeLego) WatchEvents() {
 
 	kl.Log().Debugf("start watching ingress objects")
 
-	resyncPeriod := 60 * time.Second
+	resyncPeriod := 10 * time.Minute
```

Comment:
Is this necessary? Setting this also to 10 minutes might cause confusion....this isn't what enforces the 1 req / 10 minute behaviour....the queue does that, so this being 1 minute shouldn't matter.
And as long as the informer and its API watch is working properly, this shouldn't need to be set so low.